pbft/test/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(test_srcs pbft_test.cpp pbft_operation_test.cpp)
+set(test_libs pbft protobuf bootstrap)
+
+add_gmock_test(pbft_test)
```

Comment:
remove _test from these. It's now added for you.

Reply:
Your tests also have a lot of gmock warnings. Please have a look to clean that up.